doc/nrf/includes/matter_sample_wifi_flash.txt

```diff
@@ -0,0 +1,27 @@
+In the Matter over Wi-Fi variant of this sample, the :ref:`a portion of the application code related to the nRF70 Series' Wi-Fi firmware <ug_nrf70_developing_fw_patch_ext_flash>` is stored in external memory.
+When you flash the sample onto your development kit, the build process automatically includes the Wi-Fi firmware patch and programs it into the external memory.
+
+By default, using the ``west flash`` command will erase the entire external flash memory before writing the new Wi-Fi firmware patch.
+This full-chip erase ensures that the patch is written to a clean memory area, but it can significantly increase flashing time—sometimes taking several minutes, especially if the external flash is large.
+
+To optimize the flashing process and reduce the time required, you can instruct ``west flash`` to erase only the specific memory ranges that will be overwritten by the Wi-Fi firmware patch.
+This is done by adding the ``--ext-erase-mode=ranges`` argument to the command.
+With this option, only the necessary sections of external memory are erased, making the flashing process much faster.
+
+For example, to flash the sample and minimize erase time, use the following command:
+
+.. code-block:: console
+
+    west flash --ext-erase-mode=ranges
+
+A longer flashing process is also visible when using the ``Erase and Flash to Board`` option in the |nRFVSC|.
+To speed up the flashing process, you can use the ``Flash`` button instead of ``Erase and Flash to Board`` in the ``Actions`` menu.
```

Comment:
```suggestion
To speed up the flashing process, you can use the :guilabel:`Flash` button instead of :guilabel:`Erase and Flash to Board` in the :guilabel:`Actions` menu.
```